json-rpc/types/src/views.rs

```diff
@@ -356,6 +358,15 @@ pub enum EventDataView {
         created_address: AccountAddress,
         role_id: u64,
     },
+    #[serde(rename = "diemiddomain")]
+    DiemIdDomain {
```

Comment:
I thought you want to add events in next PR. You may also add this data structure doc if you want to keep this change